tests/js/framework/modules/layout/testArticleStack.js

```diff
@@ -95,7 +95,7 @@ describe('Layout.ArticleStack', function () {
         selection.get_models.and.returnValue([different_article_model]);
         selection.emit('models-changed');
         Utils.update_gui();
-        let new_view = factory.get_created('contents.article.1')[0];
+        let new_view = factory.get_created('contents.article')[1];
```

Comment:
Same as above, the moz60tool does not report this as an error. I will run the test suit to see if this change is needed to adapt the test to the changes.

Reply:
Yes, after running `flapjack test eos-knowledge-lib` I can see that's the case. Looks good to merge.